src/components/settings/SettingsLMSTab/ErrorReporting/SyncHistory.jsx

```diff
@@ -230,6 +230,9 @@ const SyncHistory = () => {
                   src={channelMapping[configChannel].icon}
                 />
                 {config.displayName}
+                {getStatus(config) !== 'Active' && (
+                  <Badge className="card-status-badge" variant="light">{getStatus(config)}</Badge>
```

Comment:
nit: I know this was an existing issue in this file, but `getStatus(config)` is used now 7 times.  Can we memo-ize this value once in this component?